docs/pages/reference/config.mdx

```diff
@@ -3,64 +3,82 @@ title: Teleport Configuration Reference
+reference file is shown below. This provides comments and all available options
+for `teleport.yaml`. 
 
-Teleport uses the YAML file format for configuration. A full configuration reference
-file is shown below, this provides comments and all available options for `teleport.yaml`
-By default, it is stored in `/etc/teleport.yaml`.
+By default, the `teleport` process reads its configuration from
+`/etc/teleport.yaml`.
 
-<Admonition type="danger" title="Before using this reference">
-<Tabs>
-<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+## Before using this reference
 
-- **Do not use this example configuration in production.**
+<Notice type="danger">
 
-  You must edit your configuration file to meet the needs of your environment.
-  Using a copy of the reference configuration will have unintended effects. To
-  create a configuration file that you can use as a starting point, run the
-  following command:
+Do not use this example configuration in production.
+
+</Notice>
+
+You must edit your configuration file to meet the needs of your environment.
+Using a copy of the reference configuration will have unintended effects. To
+create a configuration file that you can use as a starting point, run the
+following command:
 
   ```code
   $ teleport configure -o file
   ```
 
-- You should back up your configuration file before making changes. This will
-  enable you to roll back to the previous configuration if you need to.
+<Notice type="warning">
 
-</TabItem>
-<TabItem scope={["oss", "enterprise"]} label="Teleport Cloud">
+You should back up your configuration file before making changes. This will
+enable you to roll back to the previous configuration if you need to.  
 
-- **Do not use this example configuration in production.**
+</Notice>
 
-  You must edit your configuration file to meet the needs of your environment.
-  Using a copy of the reference configuration will have unintended effects. To
-  create a configuration file that you can use as a starting point, run the
-  following command:
+## Enabling Teleport services
 
-  ```code
-  $ teleport configure -o file
-  ```
+The `teleport` process can run multiple services. 
 
-- You should back up your configuration file before making changes. This will
-  enable you to roll back to the previous configuration if you need to.
+For some services, you must enable the service within your Teleport
+configuration in order to start it. Other services are enabled by default. 
 
-- Teleport Cloud manages the Auth Service and Proxy Service for you. Your
-  Teleport resource services (e.g., the Application Service and Database
-  Service) should include the following configuration options to avoid
-  unintended effects:
+To enable or disable a service, include the following in your Teleport
+configuration, replacing `service_name` with the name of your service (service
+names are listed below):
 
-  ```yaml
-  auth_service:
-    enabled: no
+```yaml
+service_name:
+  enabled: false
```

Comment:
Does the teleport configure command output true/false or yes/no? It would be nice to make sure this matches.